### DISCUSSION TITLE
I am confused what is the core difference between RGB20 and RGB25? Why do we need both of them?

### DISCUSSION THREAD

**Discussion by saltduck** [RGB-WG/rgb#90] [Q&A] [answered]

**Comment by dr-orlovsky** [comment, marked as answer]
RGB20 is like ERC20: fungible single token.

RGB25 is a mid-ground between a fungible token and an NFT: like fungible, it can be fractional - and like in NFT some non-fungible asset is associated with it. The information about such asset is provided in the `contractData` global state, which doesn't exist in RGB20. Unlike RGB20, for obvious reasons, RGB25 tokens can't have any inflation/secondary issue (since the backing non-fungible asset must fully exist at the moment of the initial issue).

A good example of RGB25 can be a share (measured in squared meters) in some fixed real estate property. The backing real estate object is non-fungible; however squared meters are fungible between themselves within the object's scope. Also, one can't "inflate" the size of the property after it is tokenized. Clearly, neither RGB20 nor RGB21 (NFTs) doesn't fit that use case, and that's why we have created RGB25.
